Fix: add back the custom rpc node setting

Zir0h opened the pull request:
I deleted this the last rpc node update, I thought this was a placeholder. I was never able to use a custom rpc node without adapting it myself in localstorage. I probably missed something obvious :pray:

Zir0h committed:
Fix: add back the custom rpc node setting

melMass commented:
The idea was to show an extra text input when custom is selected but it doesn't seem to be in either main or develop so it's probably something I only did locally... I can't recall.
I'll add it tonight or tomorrow to this PR

melMass committed:
fix: ✨ make use of the custom rpc node

Not tested yet but the logic is there :)

melMass committed:
fix: ✨ hacky way to fix custom rpc input

Ideally the whole settings should be a form that supports validation.
We have the component but it should be a separate PR imo.
The other option I added a todo for is having custom validation / input
which should be quite easy to add to the existing one, it should be
done from within onBlur while still supporting a custom onBlur.

melMass committed:
fix: ✨ Skip display "hide banner" if no active banner

melMass committed:
fix: ✨ Apply auto audit fix

melMass committed:
fix: ✨ rebuild the lock file from scratch

melMass committed:
Merge branch 'main' into fix-add-custom-rpc-node

melMass committed:
Merge branch 'main' into fix-add-custom-rpc-node

melMass committed:
Merge branch 'main' into fix-add-custom-rpc-node

melMass reviewed (approved):
I'll merge in a few minutes 💌
